control-station/src/services/PodSocketClient.ts

```diff
@@ -5,21 +5,14 @@ import { ioNamespace } from "./socketHandler";
 interface ServerToClientEvents {
 	connect: () => void;
 	disconnect: (reason: Socket.DisconnectReason) => void;
-	pong: (data: string) => void;
-	greet: (data: string) => void;
-	stop: (data: string) => void;
-	forcestop: (data: string) => void;
-	load: (data: string) => void;
-	start: (data: string) => void;
+	serverResponse: (data: string) => void;
 }
 
 interface ClientToServerEvents {
-	ping: (data: string, ack: (data: string) => void) => void;
-	greet: (data: string, ack: (data: string) => void) => void;
-	stop: (data: string, ack: (data: string) => void) => void;
-	halt: (data: string, ack: (data: string) => void) => void;
-	load: (data: string, ack: (data: string) => void) => void;
-	run: (data: string, ack: (data: string) => void) => void;
+	stop: (ack: (data: string) => void) => void;
+	halt: (ack: (data: string) => void) => void;
+	load: (ack: (data: string) => void) => void;
+	run: (ack: (data: string) => void) => void;
```

Comment:
Nitpick: while we're at it, could we also reorder these (and the corresponding emitters and server-side handlers) to reflect a "standard" progression of events: load, run, stop, halt?